examples/getting_started/1-Introduction.ipynb

```diff
@@ -200,7 +201,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "HoloViews supports ``numpy``, ``xarray``, ``iris``, and ``dask`` arrays when working with array data (see [Gridded Datasets](../user_guide/Gridded_Datasets.ipynb)).  We can also compose elements containing array data with those containing tabular data. To illustrate, let's pass our tabular station data to a [``Points``](../reference/elements/bokeh/Points.ipynb) element which is used to mark positions in two-dimensional space:"
+    "HoloViews supports ``numpy``, ``xarray``, ``iris``, and ``dask`` arrays when working with array data (see [Gridded Datasets](4-Gridded_Datasets.ipynb)).  We can also compose elements containing array data with those containing tabular data. To illustrate, let's pass our tabular station data to a [``Points``](../reference/elements/bokeh/Points.ipynb) element, which is used to mark positions in two-dimensional space:"
```

Comment:
Let's drop iris here, it's moved to geoviews and seems pretty niche anyway.

Reply:
Fine by me!

Reply:
Fixed in f2ce0219